vespalib/src/tests/stllike/asciistream_test.cpp

```diff
@@ -481,6 +484,59 @@ AsciistreamTest::testDouble() {
     VERIFY_DOUBLE_SERIALIZATION(0.0, "0.0", automatic << forcedot, 1);
     VERIFY_DOUBLE_SERIALIZATION(0.0, "0.0", automatic << forcedot, 16);
     VERIFY_DOUBLE_SERIALIZATION(maxInteger, "9007199254740992.0", automatic << forcedot, 16);
+
+    asciistream as;
+    as.clear();
+    as << (3 * std::numeric_limits<double>::min());
+    double dv = 0;
+    as >> dv;
+    EXPECT_TRUE(dv > 0);
+
+    as.clear();
+    as << (3 * std::numeric_limits<double>::denorm_min());
+    dv = 0;
+    as >> dv;
+    EXPECT_TRUE(dv > 0);
+
+    as.clear();
+    as << "1.0e-325";
+    dv = 42.0;
+    as >> dv;
+    EXPECT_EQUAL(dv, 0.0);
+
+    as.clear();
+    as << "1.0e666";
+    dv = 42.0;
+    EXPECT_EXCEPTION(as >> dv, IllegalArgumentException, "double value '1.0e666' is outside of range.");
+    EXPECT_EQUAL(dv, 42.0);
+}
+
+void
+AsciistreamTest::testFloat() {
```

Comment:
consider doing float test the same way as double test

Reply:
done